client/my-sites/customer-home/cards/features/help-search/index.jsx

```diff
@@ -0,0 +1,103 @@
+/**
+ * External dependencies
+ */
+import { Card } from '@automattic/components';
+import React, { useEffect } from 'react';
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import CardHeading from 'components/card-heading';
+import Gridicon from 'components/gridicon';
+
+import { getSearchQuery } from 'state/inline-help/selectors';
+import { hideInlineHelp, showInlineHelp } from 'state/inline-help/actions';
+import { openSupportArticleDialog } from 'state/inline-support-article/actions';
+import QuerySupportTypes from '../../../../../blocks/inline-help/inline-help-query-support-types';
+import HelpSearchCard from './search-card';
+import HelpSearchResults from './search-results';
+
+import { RESULT_POST_ID, RESULT_LINK } from '../../../../../blocks/inline-help/constants';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const amendYouTubeLink = ( link = '' ) =>
+	link.replace( 'youtube.com/embed/', 'youtube.com/watch?v=' );
```

Comment:
Legacy due to copy/paste. Looks like we'd be ok to remove.

Reply:
This comes from:

https://github.com/Automattic/wp-calypso/blob/c57c7a5d4b807be316b200f586321def6d70e381/client/blocks/inline-help/inline-help-rich-result.jsx#L31-L32

It looks like it's because some support search results can be links to YouTube videos. That said I'm not sure these would show up on the Home page. I've not been able to find a single one yet.